### DISCUSSION TITLE
Bloop Animation iPad Course

### DISCUSSION THREAD

**Discussion by ota-works** [keiwando/toonsquid#389] [General]
Hiya, everyone! 
I hope you're all doing great. I came across an announcement for an upcoming iPad animation course back in late July, and wanted to inform Keiwan, but I got busy. I finally remembered to share this info not only with him, but with everyone else here.
The special info doesn't show on the website (https://www.bloopanimation.com/ipad-animation/), but in my subscribed mail, I came across this;
"
So what is this course about?
In a few words - this course covers EVERYTHING you need to animate on an iPad, for every skill level.

From simple short animations to advanced rigged characters.

We do that in this course by taking you through animating an entire animated short, and breaking the process into different programs, depending on the complexity needed.

We'll start with basic visual development using Procreate, and move on to drawing character sheets and complete storyboards.

We'll then turn these storyboards into an animatic using DaVinci Resolve for editing, and Garage Band for audio. All on the iPad.

We'll then dive into our first piece of animation - a quick shot using Procreate. This will be our first go at animating on an iPad, so we'll keep it simple (for now).

We'll then move on to FlippaClip for more advanced frame-by-frame animation, including lip sync.

Then, for the final shot, we'll go into ToonSquid, for the complete animation experience on an iPad, going over raster/vector drawing tools, advanced frame-by-frame animation, layer animation, camera motion, using symbols, and even rigging a character.

We'll end the course by compositing and editing our short in DaVinci Resolve, and exporting it for the final piece.

This course is one of the most exciting ones we've made so far.

It shows the endless possibilities of animating with just an iPad and a stylus, anywhere you are, whenever you feel like it.

And if you never animated before, it will definitely get you excited to try.

​[The course] will come out sometime in September, so stay tuned!
"

They are gonna be using Toonsquid for the course!!! 
I hope you all are as stoked as I was when I first came across it and still am to this day. 

It's all very exciting!!! 
Congrats, Keiwan!

**Comment by DWrightAnimation** [comment]
Too bad they didn't give any details of what's in the course, but I'm glad they're taking the steps to encourage people with only an iPad and no computer to give animation a go. Hope ToonSquid is included or mentioned in the course.

**Reply by ota-works** [reply]
"Then, for the final shot, we'll go into ToonSquid, for the complete animation experience on an iPad, going over raster/vector drawing tools, advanced frame-by-frame animation, layer animation, camera motion, using symbols, and even rigging a character."

They already said they'll be using Toonsquid as one of the apps they animate with. 😄

**Reply by DWrightAnimation** [reply]
Looked into it, found a more detailed article from Bloop themselves, and you're right. It'll use Procreate, DaVinci Resolve, GarageBand, Flippaclip, and ToonSquid.
